test/controllers/users.js

```diff
@@ -99,5 +106,52 @@ describe('users controller', () => {
         });
     });
   });
+  describe('/users/sessions/renew POST', () => {
+    it('should fail because header is not being sent', (done) => {
+      chai.request(server)
+        .post('/users/sessions/renew')
+        .send()
+        .catch((err) => {
+          err.response.should.be.json;
+          err.response.body.error.should.equal('User is not logged in');
+          err.should.have.status(401);
+        }).then(() => {
+          done();
+        });
+    });
+    it('should fail because renew_id is not being sent', (done) => {
+      successfulLogin().then((res) => {
+        chai.request(server)
+          .post('/users/sessions/renew')
+          .set(sessionManager.HEADER_NAME, res.headers.authorization)
+          .send()
+          .catch((err) => {
+            err.response.body.error.should.equal('Invalid renewId');
+            err.response.should.be.json;
+            err.should.have.status(400);
+          }).then(() => {
+            done();
+          });
+      }).catch((err) => {
+        done(err);
+      });
+    });
+    it('should be successful getting a renewed id', (done) => {
+      successfulLogin().then((res) => {
+        chai.request(server)
+          .post('/users/sessions/renew')
+          .set(sessionManager.HEADER_NAME, res.headers.authorization)
+          .send({ renew_id: res.headers.renewid })
+          .then((response) => {
+            response.should.be.status(200);
```

Comment:
We could check that the jwt, renewId and expirationDate are different from the previous ones.